step-capstone/src/Components/TravelObject.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react'
 import { Box, Grid, Typography, Card, CardContent } from '@material-ui/core'
 import { Edit, Delete } from '@material-ui/icons'
+import Hotel from './Hotel'
 
 export default function TravelObject(props) {
     let content = null;
-    switch (props.type) {
+    switch (props.data.type) {
         case 'event':
             content = <Typography variant="h4" gutterBottom>Event!</Typography>
             break;
         case 'flight':
             content = <Typography variant="h4" gutterBottom>Flight!</Typography>
             break;
         case 'hotel':
-            content = <Typography variant="h4" gutterBottom>Hotel!</Typography>
+            content = <Hotel {...props.data}/>
```

Comment:
What's the purpose of "..."? Would it maybe be clearer to pass it in like: data={ props.data }?

Reply:
...props.data would deconstruct props.data, so that going into Hotel the data fields are listed seperately. so in Hotel you can just do props.title. If you do data = props.data you are creating a specific prop within props that contains all the data fields. so you would need to do props.data.title. It depends on whether we think anything else other than those data fields is getting passed to hotel.